src/iceberg/CMakeLists.txt

```diff
@@ -17,11 +17,16 @@
 
 set(ICEBERG_SOURCES demo_table.cc)
 
+set(ICEBERG_AVRO_STATIC_BUILD_LIBS)
+list(APPEND ICEBERG_AVRO_STATIC_BUILD_LIBS Avro::avro_static)
```

Comment:
To fully support reading (and writing) Avro files, we may need to add `zlib` and `snappy` as dependencies. Should we add a `libiceberg-avro` instead and only define the Avro reader/writer interfaces in the `libiceberg` to make it depend on less libraries?

Reply:
Reasonable, will work on this.

Reply:
@wgtmac I have implemented as you suggested, using iceberg_arrow as a template, pls give it a look when you are available, thanks.